examples/vertx/src/main/java/example/vertx/MainVerticle.java

```diff
@@ -0,0 +1,56 @@
+package example.vertx;
+
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.Future;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.RoutingContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MainVerticle extends AbstractVerticle {
+
+    private final Logger logger = LoggerFactory.getLogger(MainVerticle.class);
+
+    @Override
+    public void start(Future<Void> startFuture) {
+        Router router = Router.router(vertx);
+
+        router.get("/").handler(this::hello);
+        router.get("/time").handler(this::now);
+
+        vertx.createHttpServer()
+                .requestHandler(router)
+                .listen(8080, ar -> {
```

Comment:
Could you use a more meaningful variable name than `ar`?  I'm not that familiar with vert.x and can't really guess what this means. `allocatedRouter`?

Reply:
`AsyncResult<T>`, we often use `ar`, but I can for sure have something more meaningful.